Stripe/PaymentMethodMessagingView.swift

```diff
@@ -0,0 +1,275 @@
+        // If the specified font is the same family as the system font, use "-apple-system" as the family name. Otherwise, the html renderer will only use the non-bold variation of the system font, breaking any bold font configurations.
+        let fontFamily = isSystemFont ? "-apple-system" : font.familyName
+        let css = """
+            <style>
+            body {
+                font-family: "\(fontFamily)";
+            }
+            </style>
+        """
+        return try await withCheckedThrowingContinuation { continuation in
+            NSAttributedString.loadFromHTML(string: css + html, options: [:]) { attributedString, attributeDict, error in
+                guard let attributedString = attributedString else {
+                    continuation.resume(with: .failure(error ?? Error.failedToInitializeAttributedString))
+                    return
+                }
+                // Set images to 3x
+                attributedString.enumerateAttribute(.attachment, in: NSRange(0..<attributedString.length)) { value, range, stop in
+                    guard
+                        let value = value as? NSTextAttachment,
+                        let imageData = value.fileWrapper?.regularFileContents,
+                        let scaledImage = UIImage(data: imageData, scale: 3)
+                    else { return }
+                    value.image = scaledImage
+                }
+                continuation.resume(with: .success(attributedString.withFontSize(font.pointSize)))
+            }
+        }
+    }
+   
+    // MARK: - Network helpers
+    struct PaymentMethodMessagingContentResponse: Decodable {
+        let display_l_html: String
+        let learn_more_modal_url: String
+    }
+    
+    static func loadContent(configuration: Configuration) async throws -> PaymentMethodMessagingContentResponse {
+        let parameters = Self.makeMessagingContentEndpointParams(configuration: configuration)
+        var request = configuration.apiClient.configuredRequest(for: APIEndpoint, additionalHeaders: [:])
+        request.stp_addParameters(toURL: parameters)
+        return try await withCheckedThrowingContinuation { continuation in
+            configuration.apiClient.get(
+                url: APIEndpoint,
+                parameters: parameters) { (result: Result<PaymentMethodMessagingContentResponse, Swift.Error>) in
+                    continuation.resume(with: result)
+                }
+        }
+    }
+    
+    static func makeMessagingContentEndpointParams(configuration: Configuration) -> [String: Any] {
+        let logoColor: String
+        switch isDarkMode() ? configuration.imageColor.userInterfaceStyleDark : configuration.imageColor.userInterfaceStyleLight {
+        case .light:
+            logoColor = "white"
+        case .dark:
+            logoColor = "black"
+        case .color:
+            logoColor = "color"
+        }
```

Comment:
Optional: 
I would put this in a function in the enum instead of here, but either way is fine.